### DISCUSSION TITLE
Storage transformations

### DISCUSSION THREAD

**Discussion by edwinpgm** [supabase/supabase#11344] [Questions]
Hi all! I am trying to use storage transformations, but it is not working from me side. I followed this guide (https://supabase.com/docs/guides/storage/image-transformations#get-a-public-url-for-a-transformed-image).

Are there some configurations that I need to do in Supabase to this work?

**Comment by kiwicopple** [comment]
Are you on the Pro tier?

 In case you missed it:

<img width="961" alt="image" src="https://user-images.githubusercontent.com/10214025/209985610-6189252d-dd4b-4d99-9a9a-29e392aeffc1.png">

**Reply by edwinpgm** [reply]
@kiwicopple Yes, we are in Pro Tier
